src/Microsoft.DotNet.Build.Tasks/Targets/publishtest.targets

```diff
@@ -83,7 +83,7 @@
       OverwriteReadOnlyFiles="$(OverwriteReadOnlyFiles)"
       Retries="$(CopyRetryCount)"
       RetryDelayMilliseconds="$(CopyRetryDelayMilliseconds)"
-      UseHardlinksIfPossible="true">
+      UseHardlinksIfPossible="$(CreateHardLinksForCopyFilesToOutputDirectoryIfPossible)">
```

Comment:
That is, a default value for CreateHardLinksForCopyFilesToOutputDirectoryIfPossible of true when $(OS)=='Windows_NT'